### PR TITLE
allow multiple stripe payment processor on back end

### PR DESCRIPTION
**Pull request by jmcclelland**
Without this patch, you can the error "no such token" if:

 * You have two different stripe payment processors configured.
 * you are making an offline contribution
 * using the provided payment processor back end, you choose the
 non-default payment processor.

The stripe javascript submits the payment using the default payment
processor, but the PHP codes tries to charge it using the user-selected
payment processor, which leads to the error.

### CODE REVIEW

**Comment by jmcclelland** [comment]
The last commit will conflict with #3. But depending on which one you commit first I can refactor.

File: js/civicrm_stripe.js [hunk 1]
```diff
@@ -161,8 +205,9 @@
       $form = getBillingForm();
 
       // Don't handle submits generated by non-stripe processors
-      if (!$('input#stripe-pub-key').length) {
-        debugging('submit missing stripe-pub-key element');
+      if (!$('input#stripe-pub-key').length || !($('input#stripe-pub-key').val())) {
+        debugging('submit missing stripe-pub-key element or value');
+        $form.get(0).submit();
```
Comment (mattwire) [inline js/civicrm_stripe.js:210]: Could we add a comment here as to why we're calling `$form.get(0).submit();`

Comment (jmcclelland) [reply js/civicrm_stripe.js:210]: good catch - that is a mistake! I'm opening a new PR to replace this one.

**Comment by jmcclelland** [comment]
I've opened #7 to replace this one - it fixes the error you identified - and another problem I experienced while testing (always loadStripeBillingBlock on ajax changes).

**Pull request closed without merging**